Framework/Http/Responses/ViewResponse.php

```diff
@@ -106,14 +102,8 @@ protected function generate(): void
      */
     private function renderView(array $data, string $viewPath): void
```

Comment:
Načo tu bola táto funkcia a lebo vyzerá že ten kod na renderovanie tu bol 2x. keď si to už prerobil tak asi by som ten output buffering presunul sem, a táto funkcia by vrátila vyrenderovaný string